test/test_other.py

```diff
@@ -8481,7 +8493,28 @@ def test_run_order(self):
       foo();
     }
     ''')
-    self.do_runf('src.c', 'addAtInit\nctor\naddAtPostCtor\nmain\n', emcc_args=['--js-library', 'lib.js'])
+    expected_order = [
+      'addOnPreRun',
+      'addAtPreRun',
+      'addOnInit',
+      'addAtInit',
+      'ctor',
+      'addOnPostCtor',
+      'addAtPostCtor',
+      'addOnPreMain',
+      'addAtPreMain',
+      'main',
+      'addOnExit',
+      'addAtExit',
+      'addOnPostRun',
+      'addAtPostRun',
+    ]
+    if self.get_setting('MINIMAL_RUNTIME'):
+      # Remove postRun events since they will not happen with MINIMAL_RUNTIME +
+      # EXIT_RUNTIME since it throws an exception during _proc_exit.
```

Comment:
I think that is only true when `exit()` is called.  When main returns normally (as in this test) I don't think anything get thrown.

Reply:
Indeed when I build the minimal runtime version of this test I don't see `proc_exit` in the generated JS file.

Reply:
I'm seeing something different. When I use `EMCC_DEBUG=1 test/runner other.test_run_order_minimal_runtime` `src.js` contains _proc_exit that throws.

Reply:
Are you building with '-sEXIT_RUNTIME'?